pkg/smartcontract/manifest/manifest.go

```diff
@@ -135,3 +137,189 @@ func (m *Manifest) DecodeBinary(r *io.BinReader) {
 		r.Err = err
 	}
 }
+
+// ToStackItem converts Manifest to stackitem.Item.
+func (m *Manifest) ToStackItem() (stackitem.Item, error) {
```

Comment:
I think `ToStackItem()` never actually returns error here and below.

Reply:
Error is possible during `Extra` jsonisation. For the rest of cases I removed error.